Sources/SpeziOpenAI/MessageView.swift

```diff
@@ -67,27 +63,27 @@ public struct MessageView: View {
         }
     }
     
+    public enum Defaults {
+        static let hideMessagesWithRoles: Set<Chat.Role> = [.system, .function]
+    }
     
-    /// - Parameters:
-    ///   - chat: The chat message that should be displayed.
-    ///   - hideSystemMessages: If system messages should be hidden from the chat overview.
-    public init(_ chat: Chat, hideSystemMessages: Bool = true) {
+    public init(_ chat: Chat, hideMessagesWithRoles: Set<Chat.Role>) {
```

Comment:
You can use default arguments like this to ensure that you don't have to pass this argument in all the other views: 
```suggestion
    public init(_ chat: Chat, hideMessagesWithRoles: Set<Chat.Role> = MessageView.Defaults.hideMessagesWithRoles) {
```

This also means that these changes are no breaking changes for SpeziML, ensuring that we can tag a new semantic version minor release and don't need to manually update all projects that use SpeziML.